fix errors with newer automake:

gmodule/Makefile.am:44: testgmodule_LDFLAGS must be set with `=' before using `+='

Makefile.am:73: BUILT_SOURCES multiply defined in condition TRUE ...
Makefile.am:11: ... `BUILT_SOURCES' previously defined here

--- Makefile.am
+++ Makefile.am
@@ -70,7 +70,7 @@
 
 CONFIGURE_DEPENDENCIES = acglib.m4		
 
-BUILT_SOURCES = stamp-gc-h #note: not glibconfig.h
+BUILT_SOURCES += stamp-gc-h #note: not glibconfig.h
 glibconfig.h: stamp-gc-h
 	@:
 stamp-gc-h: config.status
--- gmodule/Makefile.am
+++ gmodule/Makefile.am
@@ -41,7 +41,7 @@
 libgplugin_b_la_LIBADD = @G_MODULE_LIBS@  $(libglib)
 
 noinst_PROGRAMS = testgmodule
-testgmodule_LDFLAGS += @G_MODULE_LDFLAGS@
+testgmodule_LDFLAGS = @G_MODULE_LDFLAGS@
 testgmodule_LDADD = libgmodule.la $(libglib) @G_MODULE_LIBS@
 
 .PHONY: files release
